cmd/geth/blsaccountcmd.go

```diff
@@ -387,13 +369,35 @@ func blsAccountImport(ctx *cli.Context) error {
 	if len(keyfile) == 0 {
 		utils.Fatalf("The keystore file must be given as argument.")
 	}
-	keyJSON, err := os.ReadFile(keyfile)
+	keyInfo, err := os.ReadFile(keyfile)
 	if err != nil {
 		utils.Fatalf("Could not read keystore file: %v", err)
 	}
 	keystore := &keymanager.Keystore{}
-	if err := json.Unmarshal(keyJSON, keystore); err != nil {
-		utils.Fatalf("Could not decode keystore file: %v.", err)
+	var accountPassword string
+	if err := json.Unmarshal(keyInfo, keystore); err != nil {
+		secretKey, err := bls.SecretKeyFromBytes(common.FromHex(strings.TrimRight(string(keyInfo), "\r\n")))
+		if err != nil {
+			utils.Fatalf("keyFile is neither a keystore file or include a valid BLS12-381 private key: %v.", err)
+		}
+		pubKeyBytes := secretKey.PublicKey().Marshal()
+		encryptor := keystorev4.New()
+		accountPassword = "1234567890" // no security issue, it's only used to create keystore in memory from secretKey.
```

Comment:
Can we use the BLS wallet password? There are two types of passwords: one for the BLS account and another for the BLS wallet. Why don't we use only the wallet password? Even if we import/create a new account, we can use the wallet password to encrypt it.

Reply:
accept